### PR TITLE
#159951829 Users should be able to follow each other

### PR DESCRIPTION
**Pull request by phillipseryazi**
#### What does this PR do?
Allows a user to follow or unfollow an Author 
#### Description of Task to be completed?
I want to be able to follow and or unfollow other authors.
#### How should this be manually tested?
- Log into authors haven 
- View an article and scroll to the bottom of the article
- Find the 'follow author' button and click on it
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/159951829
#### Screenshots (if appropriate)
screen shot 2018-09-17 at 12 04 46
screen shot 2018-09-17 at 12 05 36
screen shot 2018-09-17 at 12 05 49

### CODE REVIEW

**Comment by AGMETEOR** [comment]
@Seryazi-Andela Please update the followers and following to reflect the actual number. For example, in the screenshot I see you are following only @abulojoshua1, then the following stats should be 1 :)

**Comment by phillipseryazi** [comment]
its done

File: src/CSS/editProfileModal.css [hunk 1]
```diff
@@ -32,5 +29,42 @@ form {
...
+.tooltip .tooltiptext::after {
+  content: "";
+  position: absolute;
+  top: 50%;
+  right: 100%;
+  margin-top: -5px;
+  border-width: 5px;
+  border-style: solid;
+  border-color: transparent black transparent transparent;
+}
```
Comment (geofrocker) [inline src/CSS/editProfileModal.css]: missing EOF

File: src/tests/containers/FollowGrid.test.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      const list = [
+        { username: 'usernamex', email: 'imagex' },
+        { username: 'usernameq', email: 'imageq' },
+      ];
+      moxios.stubRequest('https://authors-haven-tabs.herokuapp.com/api/users/my/following/', {
+        status: 200,
+        response: { list },
+      });
+      const spy = jest.spyOn(wrapper.instance(), 'handleResponse');
+      console.log('>>>>>', wrapper.instance().fetchData());
```
Comment (geofrocker) [inline src/tests/containers/FollowGrid.test.js]: Remove console.log statements

File: src/tests/containers/FollowerGrid.test.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+      const list = [
+        { username: 'usernamex', email: 'imagex' },
+        { username: 'usernameq', email: 'imageq' },
+      ];
+      moxios.stubRequest('https://authors-haven-tabs.herokuapp.com/api/users/my/followers/', {
+        status: 200,
+        response: { list },
+      });
+      const spy = jest.spyOn(wrapper.instance(), 'handleResponse');
+      console.log('>>>>>', wrapper.instance().fetchData());
```
Comment (geofrocker) [inline src/tests/containers/FollowerGrid.test.js]: same here

**Review by geofrocker — CHANGES_REQUESTED** [review]

File: src/tests/containers/FollowerGrid.test.js (hunk 3 above)
Comment (phillipseryazi) [reply src/tests/containers/FollowerGrid.test.js]: let me work on all

**Comment by lytes20** [comment]
When I try to read an article on this PR's app, could please find out what is causing this! @Seryazi-Andela 
screen shot 2018-09-18 at 18 13 17

**Comment by ssewilliam** [comment]
@lytes20,
- This is a bug that is not for this PR. 
- This is coming in because the article was deleted
- The bug might be corrected later this week.

**Comment by lytes20** [comment]
@ssewilliam thank you for the clarification

**Review by lytes20 — APPROVED** [review]

File: src/components/Profile/PersonCard.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  };
+
+  resetButtonText = () => {
+    this.setState({
+      buttonText: 'Following',
+    });
+  };
+
+  onFocus = () => {
+    // console.log('unfollow clicked');
```
Comment (ssewilliam) [inline src/components/Profile/PersonCard.js]: @Seryazi-Andela 
- this is another commented line

File: src/components/Profile/FollowerCard.js [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+  };
+
+  resetButtonText = () => {
+    this.setState({
+      buttonText: 'Follower',
+    });
+  };
+
+  onFocus = () => {
+    // console.log('unfollow clicked');
```
Comment (ssewilliam) [inline src/components/Profile/FollowerCard.js]: @Seryazi-Andela 
- This is commented code.

**Review by ssewilliam — COMMENTED** [review]
@Seryazi-Andela 
- There are two more commented lines that you have not taken out.

**Review by ssewilliam — APPROVED** [review]

**Pull request merged**